BackEndFlask/controller/Routes/Course_routes.py

```diff
@@ -4,17 +4,38 @@
 from flask_jwt_extended import jwt_required
 from controller.security.customDecorators import AuthCheck, badTokenCheck
 from models.course import(
-    get_courses_by_admin_id, get_courses, get_course, 
-    create_course, replace_course
+    get_courses_by_admin_id,
+    get_courses,
+    get_course, 
+    create_course,
+    replace_course
 )
 
 @bp.route('/course', methods = ['GET'])
 @jwt_required()
 @badTokenCheck()
 @AuthCheck()
-def get_all_courses():
-    if request.args and request.args.get("admin_id"):
-        admin_id = request.args.get("admin_id")
+def get_courses():
+    # TODO: Return courses that:
+    # [x] an Admin owns.
```

Comment:
Comment for future: Would it work if the route returned all courses a person is enrolled in and the persons' relationship to that course? Since role is not a property of a person, this route would need to be given a user id and a role id to fulfill the requirement as currently written. If we simply got all the courses and handed them back with the role ids, the frontend could then display them differently. Might not be best if we frequently want all the courses where someone is an TA but not the ones where they are a student.

Reply:
I have solved the problem and have implemented a solution in SKIL-174! I have written and updated the utility function get_courses_by_user_courses_by_user_id() that retrieves all courses and corresponding role_ids and returns them!